docs/source/index.rst

```diff
@@ -28,20 +28,16 @@ If you notice any issues, please let us know `on GitHub <https://github.com/ihme
 Introduction
 ------------
 
-The University of Washington IHME Simulation Science Team is excited to introduce pseudopeople, the Python package that simplifies Entity Resolution (ER) research and development.
-This package generates large-scale, simulated population data according to specifications by the user, to replicate a range of complexities of real applications of probabilistic record linkage software.
+The University of Washington IHME Simulation Science Team is excited to introduce pseudopeople, 
```

Comment:
I think we also wanted to make sure we define IHME somewhere, but I'm not sure of the best way to do it.